packages/rendermime/src/outputmodel.ts

```diff
@@ -345,9 +345,7 @@ namespace Private {
     for (let mimeType in bundle) {
       let item = bundle[mimeType];
       // Convert multi-line strings to strings.
```

Comment:
This comment can be removed now.

Reply:
Or perhaps you can change the comment to note that the purpose of the lines below is to deep-copy anything that isn't a primitive.

Reply:
Is the purpose to deep-copy or parse stringified JSON? One other question: if `item` is a number, boolean, or null value, should we `JSON.parse` that, too?

Reply:
A number, boolean, string, or null value is immutable, so doing a copy doesn't help us. Notice that we first stringify, then parse it. Basically, this is an optimized way of doing a deep copy of some structure, i.e., a deep copy of an array or dict.

Reply:
(the input here is not a string containing JSON, the input here is an already-parsed JSON structure, i.e., `item` has already been through `JSON.parse`)